src/seedsigner/controller.py

```diff
@@ -355,12 +355,12 @@ def run(self):
                 # Do not push a "new" destination if it is the same as the current one on
                 # the top of the stack.
                 if len(self.back_stack) == 0 or self.back_stack[-1] != next_destination:
-                    print(f"Appending next destination: {next_destination}")
+                    logger.info(f"Appending next destination: {next_destination}")
                     self.back_stack.append(next_destination)
                 else:
-                    print(f"NOT appending {next_destination}")
+                    logger.info(f"NOT appending {next_destination}")
                 
-                print("-" * 30)
+                logger.info("-" * 30)
```

Comment:
(fussy nit) This line separator can be eliminated. The way the logs come out after the `back_stack` dump now provide clear separation.

Reply:
removed in https://github.com/SeedSigner/seedsigner/pull/558/commits/0c9d955f3fde3ff4b2a449eb86242f216cded1a6